R/get_xronos.R

```diff
@@ -0,0 +1,36 @@
+#' @rdname db_getter_backend
+#' @export
+get_xronos <- function(db_url = get_db_url("xronos")) {
+
+  check_connection_to_url(db_url)
```

Comment:
Does this already download the whole database, because `db_url` points to the endpoint at `https://xronos.ch/api/v1/data/`? It takes forever. Probably it's sufficient to check if `https://www.xronos.ch` is up here. Or maybe the API has a more minimal endpoint to check if it's available.